scraper.rb

```diff
@@ -3,13 +3,16 @@
 # frozen_string_literal: true
 
 require 'pry'
+require 'pp'
```

Comment:
I don't think you actually need to `require` this. (Though it's also not clear why you're using `pp` rather than just a plain `puts` anyway.)